src/mame/taito/taito_f3_v.cpp

```diff
@@ -2103,15 +2105,14 @@ void taito_f3_state::scanline_draw(bitmap_rgb32 &bitmap, const rectangle &clipre
 			/* set alpha level */
 			if (alpha_level != m_alpha_level_last)
 			{
-				int a = alpha_level;
-				const int b = (a >> 8) & 0xf;
-				const int c = (a >> 4) & 0xf;
-				const int d = (a >> 0) & 0xf;
-				a >>= 12;
+				const u8 a = alpha_level >> 12;
+				const u8 b = alpha_level >> 8 & 0xf;
+				const u8 c = alpha_level >> 4 & 0xf;
+				const u8 d = alpha_level >> 0 & 0xf;
```

Comment:
Please leave the parentheses in where it makes things clearer (or use `BIT(alpha_level, 8, 4)` etc.).

Reply:
Used BIT() here.